backend/queries/user_query.py

```diff
@@ -0,0 +1,21 @@
+from models.user import User
+from database import SessionLocal
+
+db = SessionLocal()
+
+class UserQuery():
+    def getUserById(user_id):
+        user = db.query(User).filter(User.id==user_id).first()
+        return user
+    
+    def setEmail(user_id, email):
```

Comment:
Function names should be lowercase, with words separated by underscores as necessary to improve readability.

https://peps.python.org/pep-0008/#function-and-variable-names